client/post-editor/editor-term-selector/index.jsx

```diff
@@ -82,6 +84,7 @@ export default connect(
 		return {
 			postType: getEditedPostValue( state, siteId, getEditorPostId( state ), 'type' ),
 			postTerms: getEditedPostValue( state, siteId, postId, 'terms' ),
+			canEditTerms: canCurrentUser( state, siteId, 'manage_categories' ),
```

Comment:
I'm conflicted because this should really be driven by `taxonomyObject.capabilities.edit_terms` but since the user capabilities object is not all-inclusive, we may not be able to accurately detect custom capability checks.

An (admittedly untidy) alternative to this would be an approach like I took in some of the `<PostActionsEllipsisMenuEdit />` component, checking to see whether the capability type is known and otherwise defaulting to a fallback:

https://github.com/Automattic/wp-calypso/blob/57f3de7/client/my-sites/post-type-list/post-actions-ellipsis-menu/edit.jsx#L57-L61